packages/mercury/src/components/paginator/_paginator-styles.scss

```diff
@@ -0,0 +1,271 @@
+    }
+  }
+
+  &__button {
+    align-items: center;
+    color: var(--mer-icon__neutral);
+    display: flex;
+    height: var(--mer-spacing--lg);
+    justify-content: center;
+    padding: var(--mer-spacing--2xs);
+    width: var(--mer-spacing--lg);
+
+    &--disabled {
+      color: var(--mer-icon__on-disabled);
+    }
+
+    &--hover {
+      color: var(--mer-icon__primary--hover);
+    }
+  }
+}
+
+%paginator__button__selector--before {
+  background-color: currentColor;
+  block-size: var(--mer-spacing--lg);
+  content: "";
+  display: grid;
+  inline-size: var(--mer-spacing--lg);
+  mask: no-repeat center / 100% var(--button-arrow-icon);
+  -webkit-mask: no-repeat center / 100% var(--button-arrow-icon);
+  place-items: center;
+}
+
+/// @group Paginator
+/// @param {String} $paginator-selector [".paginator"] -
+/// @param {String} $paginator__go-to-selector [".paginator::part(go-to)"] -
+/// @param {String} $paginator__go-to__input-selector [".paginator::part(go-to__input)"] -
+/// @param {String} $paginator__items-per-page-selector [".paginator::part(items-per-page)"] -
+/// @param {String} $paginator__items-per-page__info-selector:
+//  [".paginator::part(items-per-page-info)"] -
+/// @param {String} $paginator__first-button-selector [".paginator::part(first__button)"] -
+/// @param {String} $paginator__last-button-selector [".paginator::part(last__button)"] -
+/// @param {String} $paginator__next-button-selector [".paginator::part(next__button)"] -
+/// @param {String} $paginator__previous-button-selector [".paginator::part(prev__button)"] -
+
+/// @param {String} $paginator__combo-box-selector [".paginator::part(items-per-page__combo-box)"] -
+/// @param {String} $paginator__combo-box--disabled-selector [".paginator[disabled]"] -
+/// @param {String} $paginator__combo-box__placeholder-selector:
+//  [".paginator[part='ch-combo-box-render--placeholder']"] -
+/// @param {String} $paginator__window-selector [".paginator::part(window)"] -
+/// @param {String} $paginator__group-selector [".paginator::part(group)"] -
+/// @param {String} $paginator__group__content-selector [".paginator::part(group__content)"] -
+/// @param {String} $paginator__group__header-selector [".paginator::part(group__header)"] -
+/// @param {String} $paginator__group__header--disabled-selector:
+//  [".paginator::part(group__header disabled)"] -
+/// @param {String} $paginator__group__header--expandable-selector:
+//  [".paginator::part(group__header expandable)"] -
+/// @param {String} $paginator__item-selector [".paginator::part(item)"] -
+/// @param {String} $paginator__item--disabled-selector [".paginator::part(item disabled)"] -
+/// @param {String} $paginator__item--nested-selector [".paginator::part(item nested)"] -
+/// @param {String} $paginator__item--selected-selector [".paginator::part(item selected)] -
+/// @param {Boolean} $add--disabled [false] -
+/// @param {Boolean} $add--placeholder [false] -
+/// @param {Boolean} $add--error [false] -
```

Comment:
The following mixin selectors are missing: 

```
$paginator__items-per-page__label-selector
```
```
$paginator__navigation-page-selector 
```

Typo in doc comment:

`$paginator__item--selected-selector` → missing a closing quote in the comment:

Current:
`@param {String} $paginator__item--selected-selector [".paginator::part(item selected)]`

Fix:
`@param {String} $paginator__item--selected-selector [".paginator::part(item selected)"]`